WMF Framework/WMFAnnouncementsFetcher.m

```diff
@@ -80,7 +80,7 @@ - (void)fetchAnnouncementsForURL:(NSURL *)siteURL force:(BOOL)force failure:(WMF
                 valid = YES;
                 *stop = YES;
             }
-            if ([cookieValue containsString:obj]) {
+            if ([header length] < 1 && [cookieValue hasPrefix:obj]) {
```

Comment:
This was in the 5.7.2 branch but apparently not merged into develop. it's the California/Canada announcement fix.